src/extra/widgets/list/lv_list.c

```diff
@@ -81,11 +81,13 @@ lv_obj_t * lv_list_add_btn(lv_obj_t * list, const char * icon, const char * txt)
     lv_obj_set_size(obj, LV_PCT(100), LV_SIZE_CONTENT);
     lv_obj_set_flex_flow(obj, LV_FLEX_FLOW_ROW);
 
+#if LV_USE_IMG == 1
     if(icon) {
         lv_obj_t * img = lv_img_create(obj);
         lv_img_set_src(img, icon);
     }
-
+#endif
+    
     if(txt) {
```

Comment:
What about adding `#if LV_USE_LABEL` here?

Reply:
I can hardly imagine a scenario where someone would use Lists without Labels (I can hardly imagine anyone using this library without labels). It'll mean only using lists with buttons, that also have no labels, i.e. empty buttons or with images. But technically possible so it's your call.

Reply:
Makes sense :slightly_smiling_face: 

The CI complains about some code formatting issues. Please run `scripts/code-format.py` (astyle needs to be installed)

Reply:
I ran it, but I don't understand, what should I correct.

Reply:
Once you run it, it should have reformatted your source code. Now you just have to run `git commit` and push it to your fork.

(If you'd rather just have me correct the formatting, I can also do that.)

Reply:
Ok, this time it worked